ceph_deploy/tests/test_cli_new.py

```diff
@@ -19,6 +21,18 @@ def test_help(tmpdir, cli):
     assert 'optional arguments' in result
 
 
+def test_bad_no_mon(tmpdir, cli):
+    with pytest.raises(cli.Failed) as err:
+        with cli(
+            args=['ceph-deploy', 'new'],
+            stderr=subprocess.PIPE,
+            ) as p:
+            result = p.stderr.read()
+    assert 'usage: ceph-deploy new' in result
```

Comment:
@alfredodeza yeah, good point.  An excellent area to clean up.  I took the multiple asserts directly from the existing tests for ``--help``, e.g. https://github.com/ceph/ceph-deploy/blob/master/ceph_deploy/tests/test_cli_install.py#L11-L13